test/unit/store.spec.js

```diff
@@ -9,9 +9,9 @@ describe('Store Unit Tests', () => {
 
   describe('init()', () => {
     it('should set computed vales', () => {
-      expect(store.unitLabel, 'to be', undefined);
+      expect(store.unitFiatLabel, 'to be', undefined);
```

Comment:
should we be testing all of the computed values, or is this just to test that `init` changes this specific value? 

i guess if we tested all of them it would be redundant to the `ComputedWallet` tests.

Reply:
Yeah this is just a smoke test of init.